app/dashboard/src/pages/authentication/Registration.tsx

```diff
@@ -74,56 +67,50 @@ export default function Registration() {
 
   return (
     <AuthenticationPage
+      schema={REGISTRATION_FORM_SCHEMA}
       title={getText('createANewAccount')}
       supportsOffline={supportsOffline}
-      footer={
-        <Link to={appUtils.LOGIN_PATH} icon={GoBackIcon} text={getText('alreadyHaveAnAccount')} />
-      }
-      onSubmit={async (event) => {
-        event.preventDefault()
-        setIsSubmitting(true)
-        await auth.signUp(email, password, organizationId)
-        setIsSubmitting(false)
+      footer={<Link to={LOGIN_PATH} icon={GoBackIcon} text={getText('alreadyHaveAnAccount')} />}
+      onSubmit={async ({ email, password, confirmPassword }): Promise<void> => {
```

Comment:
What was the reason to use manual verification logic instead of using `schema` based approach?

Reply:
the reason is i forgor about `schema` :skull: